pages/signup/app.js

```diff
@@ -8,8 +8,10 @@ import { validationUsers } from "../../utils/validationUsers.js";
 (function () {
   // states & elements
   let passwordWithEyeIconsFlag = false;
+  let alreadyEmailValidation = false;
+  let alreadyPasswordValidation = false;
```

Comment:
```suggestion
  let isEmailValidated = false;
  let isPasswordValidated = false;
```
취향의 영역이지만 좀 더 일반적인 네이밍을 추천해 봤어요 :)